src/pages/docs/octopus-rest-api/openid-connect/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-09-27
+modDate: 2023-09-27
+title: Using OpenID Connect with the Octopus API
+description: External systems can use OpenID Connect with service accounts to access the Octopus API without needing to provision API keys
+navOrder: 30
+hideInThisSection: true
+---
+
+Octopus supports using [OpenID Connect (OIDC)](https://openid.net/) to access the Octopus API without needing to provision API keys.
+
+Some of the benefits of using OIDC include:
+
+- API keys do not need to be provisioned and stored in external systems, reducing the risk of unauthorized access to the Octopus API from exposed keys.
+- Access tokens granted by Octopus are short-lived, reducing the risk of unauthorized access to the Octopus API.
+- API keys do not need to be rotated manually by administrators, reducing the risk of disruption when updating to newer keys in external systems.
```

Comment:
Its worth adding a comment/section about how the permissions provided by the token generated are based on the permissions provided by the service account.  This allows the user to configure fine-grained controls for their needs, supporting our recommended principle of least access. Something that helps provide a background of why this architecture was selected

Reply:
That's true, the same thing applies when using API keys though right (the permissions come from the service account)? So it's not necessarily a benefit of oidc as such, but yeah could still be worth mentioning, perhaps more in terms of the OIDC identity itself is granular via the specific subject string, so it promotes principle of least access from the external in (if that makes sense).